test.py

```diff
@@ -0,0 +1,125 @@
+import logging
+from pathlib import Path
+from shutil import copytree
+
+import pytest
+import kubernetes_env.util as util
+import kubernetes_env.kube_env as kube_env
+
+# configure logging
+log = logging.getLogger(__name__)
+logging.basicConfig(filename="test.log",
+                    format="%(levelname)s:%(message)s",
+                    level=logging.INFO,
+                    filemode='w')
+stderr_log = logging.StreamHandler()
+stderr_log.setFormatter(logging.Formatter("%(levelname)s:%(message)s"))
+logging.getLogger().addHandler(stderr_log)
+
+# some folder definitions
+FILE_DIR = Path.resolve(Path(__file__)).parent
+SIM_DIR = FILE_DIR.joinpath("tracing_sim")
+COMPILER_DIR = FILE_DIR.joinpath("tracing_compiler")
+ENV_DIR = FILE_DIR.joinpath("kubernetes_env")
+COMPILER_BINARY = COMPILER_DIR.joinpath("target/debug/dtc")
+QUERY_DIR = COMPILER_DIR.joinpath("example_queries")
+UDF_DIR = COMPILER_DIR.joinpath("example_udfs")
+
+KUBE_QUERIES = [
+    pytest.param("count.cql", ["count"]),
+    pytest.param("breadth_histogram.cql", ["histogram"]),
+    pytest.param("height_histogram.cql", ["histogram"]),
+    pytest.param("response_code_count.cql", ["count"]),
+    pytest.param("response_size_avg.cql", ["avg"]),
+    pytest.param("return.cql", []),
+    pytest.param("return_height.cql", []),
+]
+# test names
+KUBE_IDS = [i.values[0] for i in KUBE_QUERIES]
+
+SIM_QUERIES = [
```

Comment:
Does this accept failing queries then?  If so we should put a TODO to update this as we go along and fix them.

Reply:
Yes. They can be marked with the xfail property then the tests will show up as XFAIL in the test script.

Reply:
Good to know, thanks